### PR TITLE
Feature/#27

### PR DESCRIPTION
**Pull request by FDelporte**
New page with tips for visitors to Belgium, to be extended...

### CODE REVIEW

File: content/visit-belgium-tips.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+=== Across Belgium
+
+==== Brugge and Gent
+
+The two cities [Brugge](https://www.visitbruges.be/en) and [Gent](https://visit.gent.be/en/see-do/inquiry-desk-visit-gent) are competing as they have the same values to offer: water and boats in the city, shopping streets, historical buildings,... Maybe best to visit both so you can make an informed choice for a next return visit?
+
+==== Pairi Daiza
+
+As "Probably The Best Zoo in Europe," [Pairi Daiza](https://www.pairidaiza.eu) is a must-see for animal lovers. It's on the other side of Belgium if you are in Antwerp, but hey, it's a tiny country ;-)
```
Comment (SimonVerhoeven) [inline content/visit-belgium-tips.adoc]: Maybe better to say 100km for those with a bit of a grander idea of country sizes?

Comment (FDelporte) [reply content/visit-belgium-tips.adoc]: good remark, done 
BTW 102km according to maps ;-)

File: content/visit-belgium-tips.adoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
+= Tips When Visiting Belgium
+:jbake-type: normalBase
+:jbake-description: Travelling to Belgium for a JUG or conference? These are some of our tips...
+:jbake-priority: 1.0
+:showtitle:
+
+During a call with BeJUG members, we collected a list of exciting places to share with the Java community. We know a JUG or conference schedule allows you to fill your days with a lot of Java, but for those of you coming early, staying longer or just looking for a good local restaurant we got you covered.
```
Comment (TomCools) [inline content/visit-belgium-tips.adoc:7]: I would add a big note that "we did not get any kind of sponsorship nor deal from any of these places." 

Or something similar. To make it clear we are just posting tips, nothing sponsored.

Comment (FDelporte) [reply content/visit-belgium-tips.adoc:9]: Good remark, note is added

**Comment by SimonVerhoeven** [comment]
Any opinions on adding some "utility" links like:
https://www.velo-antwerpen.be/
https://www.slimnaarantwerpen.be/en/public-transport/tram-bus

**Comment by TomCools** [comment]
@SimonVerhoeven go for it 😊

**Review by doppleware — COMMENTED** [review]
Coffee shop:
Caffenation https://goo.gl/maps/3bsm2LQwT5yxfimN6

**Review by doppleware — COMMENTED** [review]
Coffe: Normo https://goo.gl/maps/DAqW6KSnXNcTkKv67

**Review by TomCools — APPROVED** [review]

**Pull request merged**